README.md

```diff
@@ -70,7 +76,70 @@ Topology picture:   Device 1 -- Device 2 -- Device 3
+        - `sudo docker build -t iptables_demoa ~/IoT_Sec_Gateway/docker_cont/demo_cont/iptables_demoa`
+        - `sudo docker build -t iptables_demob ~/IoT_Sec_Gateway/docker_cont/demo_cont/iptables_demob`
+ 
+  - **4) Start ODL on Data Plane**
+      - On "Dataplane", you should now see the __l2switch__ folder in root
+      - On "Dataplane", run the following command: 
+      `./l2switch/startODL.sh`
+      - If an error occurs, try running `sudo ./l2switch/build.sh` first and then rerun `./l2switch/startODL.sh`
+      - You should see a "Ready" message on the ODL console letting you know it is ready to receive ARP packets
+      
+  - **5) Test**
+      - Attempt to send 1 ping from "Node_0" to "Node_1" to create the ARP request (`ping 10.1.1.2 -c 1`)
+        - __Note:__ This make take up to 15 seconds for the next ARP packet in case we missed it 
+      - On "Dataplane", Node_0's MAC address will match with the policy (as specified in step 1) and deploy middlebox iptables_demoA
+      - Attempt to send another ping from "Node_0" to "Node_1" 
+      - On "Dataplane", you should see messages affirming a new container was started
+      - ICMP packets should now be dropped.  Use netcat to test that other packets like TCP can still be received.
+      
+## Steps for running the experiment in CloudLab using 1 snort middlebox and multiple archive files)
+_This experiment demonstrates the "archive" property of the JSON policy file and how you can transition to the same middlebox with a different set of configurations/rules.  The experiment starts when an ARP request is received by the dataplane by either "Node_0" or "Node_1" to deploy a Snort middlebox designed to log ICMP packets.  Instead of switching to another static middlebox, when an ICMP packet is received, the current middlebox logs the packet, triggers an alert and causes a transition where the middlebox deployed is the same but a new local.rules (the archive file) is being copied over to the middlebox.  This implementation requires only a few line changes in the JSON file._
+
+- **1) Initial setup**
+    - Please follow steps 1 and 2 from [above](#steps-for-running-the-experiment-in-cloudlab-using-2-snort-middleboxes).
+- **2) Further configure JSON**
+    - Open __IoT_Sec_Gateway/policies/cloudlab-NewPolicy20.json__ 
+    - Under the first device, "device0", change the __inMac__ to the MAC address of your "Node_0" or "Node_1"
+    - Look for the __archives__ section and you will notice two tar-path pairs.  The tar is the file on the controller and path               represent where it will be stored inside the middlebox
+    - Save and close.        
+ - **3) Configure middlebox files**
+    - On "Dataplane", `cd` into __IoT_Sec_Gateway/docker_containers/demo_conts/snort_base__
+    - Run the following command `sudo ./genTar.sh` to generate and move the snort rules and config file to your _/mnt/_ folder on your          controller/dataplane node.
```

Comment:
instead of using /mnt/ can we change this to /etc/IoT_Sec. And note that this directory might need to be created (or add that into the genTar.sh script)?